### ISSUE TITLE
Martin Material

### ISSUE THREAD

**Issue by mbaddar1** [mbaddar1/phd#24]
meigel  11:46 AM
pls check
http://proceedings.mlr.press/v84/izmailov18a.html
PMLRPMLR
Scalable Gaussian Processes with Billions of Inducing Inputs via Tensor Train Decomposition
We propose a method (TT-GP) for approximate inference in Gaussian Process (GP) models. We build on previous scalable GP research including stochastic variati...
Mar 31st, 2018
11:49
original NODE, examine normalizing flows application!
https://arxiv.org/abs/1806.07366
arXiv.orgarXiv.org
Neural Ordinary Differential Equations
We introduce a new family of deep neural network models. Instead of specifying a discrete sequence of hidden layers, we parameterize the derivative of the hidden state using a neural network. The...
Pinned by you ∙ Saved

meigel  12:04 PM
There are many new papers that have to be checked for interesting directions, e.g.
https://arxiv.org/abs/2002.08071
https://arxiv.org/abs/2006.07220
http://proceedings.mlr.press/v119/zhang20h.html
https://arxiv.org/abs/2002.09779
https://openreview.net/forum?id=HxzSxSxLOJZ
https://arxiv.org/abs/2006.10711
https://arxiv.org/abs/1912.05475
https://arxiv.org/abs/2007.15386
http://proceedings.mlr.press/v119/finlay20a/finlay20a.pdf
https://arxiv.org/abs/2003.08063
https://arxiv.org/abs/2103.12413
https://arxiv.org/abs/2105.14070
arXiv.orgarXiv.org
Dissecting Neural ODEs
Continuous deep learning architectures have recently re-emerged as Neural Ordinary Differential Equations (Neural ODEs). This infinite-depth approach theoretically bridges the gap between deep...
arXiv.orgarXiv.org
On Second Order Behaviour in Augmented Neural ODEs
Neural Ordinary Differential Equations (NODEs) are a new class of models that transform data continuously through infinite-depth architectures. The continuous nature of NODEs has made them...
PMLRPMLR
Approximation Capabilities of Neural ODEs and Invertible Residual Networks
Recent interest in invertible models and normalizing flows has resulted in new architectures that ensure invertibility of the network model. Neural ODEs and ...
Nov 21st, 2020
arXiv.orgarXiv.org
Stochasticity in Neural ODEs: An Empirical Study
Stochastic regularization of neural networks (e.g. dropout) is a wide-spread technique in deep learning that allows for better generalization. Despite its success, continuous-time models, such as...
OpenReviewOpenReview
ResNet After All: Neural ODEs and Their Numerical Solution
A key appeal of the recently proposed Neural Ordinary Differential Equation (ODE) framework is that it seems to provide a continuous-time extension of discrete residual neural networks. As we show...
:+1:
1

**Comment by mbaddar1** [comment]
Bayesian Optimization Book 
https://bayesoptbook.com/

**Comment by mbaddar1** [comment]
Neural Differential Equations Book
https://arxiv.org/pdf/2202.02435.pdf
